Tests/test_GenBank.py

```diff
@@ -2455,6 +2455,43 @@ def test_record_parser_tsa(self):
             record, length, locus, definition, accession, titles, features
         )
 
+    def test_record_parser_tls(self):
+        path = "GenBank/tls_KDHP01000000.gb"
+        with open(path) as handle:
+            records = GenBank.Iterator(handle, self.rec_parser)
+            record = next(records)
+        length = 0
+        locus = "KBUV01000000"
+        definition = "TLS: soil metagenome 16S ribosomal RNA, targeted locus study"
+        accession = ["KBUV00000000"]
+        titles = (
+            "Spatio-temporal dynamics of soil bacterial communities in function of Amazon forest phenology",
+            "Direct Submission",
+        )
+        features = [
+            (
+                "source",
+                "1..3714",
+                (
+                    ("/organism=", '"soil metagenome"'),
+                    ("/mol_type=", '"genomic DNA"'),
+                    (
+                        "/isolation_source=",
+                        '"soil samples in a lowland tropical evergreen rain forest in Amazonia"',
+                    ),
+                    ("/db_xref=", '"taxon:410658"'),
+                    ("/environmental_sample", ""),
+                    ("/country=", '"Brazil: Manaus"'),
+                    ("/lat_lon=", '"2.92 S 59.95 W"'),
+                    ("/collection_date=", '"2013"'),
+                    ("/note=", '"metagenomic"'),
+                ),
+            )
+        ]
+        self.perform_record_parser_test(
+            record, length, locus, definition, accession, titles, features
+        )
```

Comment:
Can you explicitly test the TLS line was read in?

(Might we worth adding something similar to the TSA test)

Reply:
Can you please elaborate on that? The only mark that this is TLS is "TLS: " in the definition at the moment.

Reply:
Also, there inside this method we are asserting length vs len(record.sequence), which is 0 for TSA and TLS. On the other hand we could use record.size for the same assertion.

Reply:
TSA and TLS have no sequence and thus no meaningful length. They do have a record count.

I mean can the test confirm ``KBUV01000001-KBUV01003714`` (from the TLS line) is available within the parsed object.

Reply:
Yes, got it, but one small detail. In current realisation we get list ['KBUV01000001', 'KBUV01003714'], and the same for TSA, shouldn't it be from the first to the last index?

Reply:
I think you are suggesting expanding this to a list of all 3714 entries or something similar (like the Python 3 range function). I was sure there was an open issue on this idea, but I can't find it right now.... context could have been broader than just GenBank / EMBL records.

Reply:
I've found this comment from you, that concerns the choice between widespread formats and language structures: https://github.com/biopython/biopython/issues/2131#issuecomment-502616697
The issue is indeed broader than this fix, so for now I'm settling on the latest update to the tests and awaiting reviews.